internal/cli/kraft/cloud/deploy/deployer_kraftfile_unikraft.go

```
@@ -62,6 +62,7 @@ func (deployer *deployerKraftfileUnikraft) Deploy(ctx context.Context, opts *Dep
 		NoUpdate:     opts.NoUpdate,
 		Platform:     "kraftcloud",
 		Rootfs:       opts.Rootfs,
+		RootfsType:   opts.RootfsType,
 		SaveBuildLog: opts.SaveBuildLog,
```

consider failing early in here because of no erofs support

---

Will add check to `Deployable` function to avoid using Unikraft if erofs